src/formatDate.js

```diff
@@ -50,7 +50,29 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newDate = date.split(`${fromFormat[fromFormat.length - 1]}`);
+  const resultDate = [];
+  const objDate = {};
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    objDate[fromFormat[i]] = newDate[i];
+  }
+
+  if (objDate.hasOwnProperty('YY')) {
+    objDate.YYYY = objDate.YY < 30
+      ? '20' + objDate.YY
+      : '19' + objDate.YY;
+  }
+
+  if (objDate.hasOwnProperty('YYYY')) {
+    objDate.YY = objDate.YYYY.slice(2);
+  }
```

Comment:
i belive you could use `if \ else if` statement here)
```suggestion
  if (objDate.hasOwnProperty('YY')) {
    objDate.YYYY = objDate.YY < 30
      ? '20' + objDate.YY
      : '19' + objDate.YY;
  } else if (objDate.hasOwnProperty('YYYY')) {
    objDate.YY = objDate.YYYY.slice(2);
  }
```